06-classification_continued.Rmd

```diff
@@ -295,14 +383,15 @@ validation set only a single time. The name for this strategy is called
 **cross-validation**.  In **cross-validation**, we split our **overall training
 data** into $C$ evenly-sized chunks, and then iteratively use $1$ chunk as the
 **validation set** and combine the remaining $C-1$ chunks 
-as the **training set**:
+as the **training set**. 
 
 
 ```{r 06-cv-image, echo = FALSE, message = FALSE, warning = FALSE, fig.cap = "5-fold cross validation", fig.retina = 2, out.width = "800"}
 knitr::include_graphics("img/cv.png")
 ```
 
-In the picture above, $C=5$ different chunks of the data set are used,
+This procedure is shown in Figure \@ref(fig:06-cv-image).
+Here, $C=5$ different chunks of the data set are used,
 resulting in 5 different choices for the **validation set**; we call this
 *5-fold* cross-validation. To do 5-fold cross-validation in R with `tidymodels`, we
 use another function: `vfold_cv`. This function splits our training data into
```

Comment:
`strata` argument is used but not explained. We might want to do CV without stratification first? And then have a section on stratification?

Reply:
clarified

Reply:
i will make an issue to maybe do stratification as its own section